CHANGELOG.md

```diff
@@ -1713,12 +1715,12 @@ v0.2.0 (2020-04-09)
   (@gotjosh)
 
   These metrics are available to monitor metadata being sent:
-    - `prometheus_remote_storage_succeeded_metadata_total`
```

Comment:
Looks like the formatter changed some values, though it looks like the new ones are more correct?

Reply:
I'm not sure what happened here, but this looks fine to me